mds/apis/prv_api/vehicles.py

```diff
@@ -70,6 +70,15 @@ class DeviceSerializer(serializers.ModelSerializer):
         help_text="Percentage between 0 and 1",
         allow_null=True,
     )
+    areas = serializers.SerializerMethodField()
+
+    def get_areas(self, obj):
+        if not obj.dn_gps_point:
+            return []
+        areas = models.Area.objects.filter(
+            polygons__geom__contains=obj.dn_gps_point
+        ).order_by("label", "id")
```

Comment:
You need to add an index on `label` in the model for this ordering to be efficient.
If you order on `label` already, I don't think ordering on `id` is really needed.

Reply:
@lionel-panhaleux `label` may be empty or null, or we should make it mandatory.

Reply:
@lionel-panhaleux Already done in previous PR for the index